This patch parially reverts the following change:

v4.2-cvs
 --------

[mms] Change 'Important' flag to 'Flagged For Followup' and clear the flag
      after a reply (Request #4867).

diff -ur imp-h3-4.3/lib/Compose.php imp-h3-4.3-important/lib/Compose.php
--- imp-h3-4.3/lib/Compose.php	2008-09-18 12:24:51.000000000 +0200
+++ imp-h3-4.3-important/lib/Compose.php	2008-10-27 17:29:09.000000000 +0100
@@ -770,13 +770,11 @@
         if (!empty($opts['reply_index']) &&
             !empty($opts['reply_type']) &&
             ($opts['reply_type'] == 'reply')) {
-            /* Make sure to set the IMAP reply flag and unset any
-             * 'flagged' flag. */
+            /* Make sure to set the IMAP reply flag. */
             require_once IMP_BASE . '/lib/Message.php';
             $imp_message = &IMP_Message::singleton();
             $idx_array = array($opts['reply_index']);
             $imp_message->flag(array('answered'), $idx_array);
-            $imp_message->flag(array('flagged'), $idx_array, false);
         }
 
         $entry = sprintf("%s Message sent to %s from %s", $_SERVER['REMOTE_ADDR'], $recipients, $_SESSION['imp']['user']);
